src/python/doc/rips_complex_ref.rst

```diff
@@ -12,3 +12,54 @@ Rips complex reference manual
    :show-inheritance:
 
    .. automethod:: gudhi.RipsComplex.__init__
+
+======================================
+Weighted Rips complex reference manual
+======================================
+
+.. autoclass:: gudhi.WeightedRipsComplex
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+   .. automethod:: gudhi.WeightedRipsComplex.__init__
```

Comment:
I find `:special-members: __init__` easier so I don't have to repeat the whole path to the class.